ReportCardParser/src/reportcardparser/ReportCardParserType1.java

```diff
@@ -1,37 +1,42 @@
-/*
- * To change this license header, choose License Headers in Project Properties.
- * To change this template file, choose Tools | Templates
- * and open the template in the editor.
- */
 package reportcardparser;
 
-/**
- * Class to handle parsing of a Type 1 report card
- * 
- * @author chris
- */
-public class ReportCardParserType1 implements IReportCardParser {
-    
-    private String filePath; // Path to the report card file
+import java.io.IOException;
+import java.io.BufferedReader;
+
+public class ReportCardParserType1 extends AbstractReportCardParser
```

Comment:
No documentation in any of the parsing classes (you even removed the comments I put there!)